addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1028,7 +1028,7 @@ options.Class.include({
                 }
         }
 
-        if (params.reload || params.noBundleReload) {
+        if (params.reload || params.noBundleReload || widgetValue?.length === 0) {
```

Comment:
1. Why not writing that `widgetValue === ""` instead of this `widgetValue?.length === 0`? :roll_eyes: 
2. Why would having an empty string value be related to not refreshing the bundle. It feels like you are fixing generically a very specific case.